### PR TITLE
[sd-4255]-(fix) - Layout issue once multiple flags are set for an item

### PR DESCRIPTION
**Pull request by sjunaid**

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk-search/react.js [hunk 1]
```diff
@@ -653,12 +644,12 @@
                                     React.createElement('span', {className: 'broadcast-status', tooltip: broadcast.status}, '!') :
                                     null,
                                 flags.marked_for_not_publication ?
-                                    React.createElement('div', {className: 'state-label not-for-publication'},
-                                        gettext('Not For Publication')) :
+                                    React.createElement('div', {className: 'state-label not-for-publication',
+                                        title: gettext('not for publication')}, gettext('Not For Publications')) :
```
Comment (petrjasek) [inline scripts/superdesk-search/react.js]: would be better to have same text for both text and title, so there is only 1 string to translate, not 2

File: scripts/superdesk-search/react.js [hunk 2]
```diff
@@ -653,12 +644,12 @@
...
                                     null,
                                 flags.marked_for_legal ?
-                                    React.createElement('div', {className: 'state-label legal'}, gettext('Legal')) :
-                                    null,
+                                    React.createElement('div', {className: 'state-label legal', title: gettext('legal')},
```
Comment (petrjasek) [inline scripts/superdesk-search/react.js]: same thing here, `'legal'` should be `'Legal'`

**Comment by mdhaman** [comment]
squash commits.

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**